js/docEnums.js

```diff
@@ -75,6 +75,11 @@
  * @enum {'alternate'|'blend'|'extrapolate'}
  */
 
+/**
+ * @typedef {string} Enums.PaletteColorSet
```

Comment:
Would it be better to call it `VizPaletteColorSet` like the others VizPalette and VizPaletteExtensionMode?